addons/userconfig/cba_settings.sqf

```diff
@@ -22,12 +22,56 @@ force ace_hearing_enableCombatDeafness = false;
 
 force ace_map_mapShowCursorCoordinates = true;
 
-force ace_medical_ai_enabledFor = 0;                      // Medical system not enabled for AI
-force ace_medical_feedback_painEffectType = 2;            // Pain effect type is "Chromatic Abberation".
-force ace_medical_gui_enableMedicalMenu = 1;              // Enable medical menu.
-force ace_medical_playerDamageThreshold = 2;              // The amount of damage a player unit can take before dying.
+force ace_medical_ai_enabledFor = 2;                      // Medical AI is Enabled for all clients and server
+force ace_medical_AIDamageThreshold = 0.8;                // The amount of damage an AI unit can take before dying.
+force ace_medical_bleedingCoefficient = 1.2;              // Coefficient for controlling the bleeding speed
+force ace_medical_blood_bloodLifetime = 900;              // Lifetime of blood drop objects
+force ace_medical_blood_enabledFor = 2;                   // Blood drops enabled for all units
+force ace_medical_blood_maxBloodObjects = 300;            // Maximum number of blood drop objects that can be spawned
+force ace_medical_fatalDamageSource = 2;                  // Either large hits to vital organs or sum of trauma can be fatal
+force ace_medical_feedback_bloodVolumeEffectType = 2;     // Low blood volume effect is Icon + Color Fading
+force ace_medical_feedback_painEffectType = 1;            // Pain effect type is "Pulsing Blur"
+force ace_medical_fractureChance = 0.3;                   // The probability of a fracture causing wound resulting in a fracture
+force ace_medical_fractures = 1;                          // Splints Fully heal fractures
+force ace_medical_gui_enableMedicalMenu = 2;              // Enable medical menu.
```

Comment:
`2` means `VehiclesOnly`, should be `1`?

Reply:
Dang, you're right. 2 is AFI setting, but let's keep it at 1